docs/src/main/sphinx/release/release-427.md

```diff
@@ -0,0 +1,108 @@
+# Release 427 (26 Sep 2023)
+
+## General
+
+* Add support for comparing IPv4 addresses and IPv6 CIDRs with the `contains`
+  function. ({issue}`18497`)
+* Improve performance of `GROUP BY`, `DISTINCT`, and other aggregations. ({issue}`19059`)
+* Fix failure due to exceeding node memory limits with `INSERT` statements. ({issue}`18771`)
+* Fix query hang for certain `LIKE` patterns involving a mix of `%` and `_`. ({issue}`19146`)
+
+## Security
+
+* Ensure access control is checked when accessing table comments. ({issue}`18514`)
+
+## Delta Lake connector
+
+* Add support for reading tables with
+  [deletion vectors](https://docs.delta.io/latest/delta-deletion-vectors.html). ({issue}`16903`)
```

Comment:
```suggestion
  [Deletion Vectors](https://docs.delta.io/latest/delta-deletion-vectors.html). ({issue}`16903`)
```

Reply:
I don't think this is correct - they're lowercase in all Delta Lake documentation.

Reply:
Both are fine. They use "Deletion Vectors" in their protocol page. https://github.com/delta-io/delta/blob/master/PROTOCOL.md#deletion-vectors